WireMessaging/Sources/WireMessagingUI/WireCells/Components/AttachmentsPreviewView/WireCellsAttachmentsPreviewItemViewModel.swift

```diff
@@ -173,6 +173,21 @@ final class WireCellsAttachmentsPreviewItemViewModel: ObservableObject {
         previewSize?.width as? Double // Explicit conversion necessary due to compiler bug
     }
 
+    var attachmentDuration: String? {
+        let fileType = attachment.contentType.flatMap { UTType(mimeType: $0) }
+
+        guard fileType == .video || fileType == .audio else {
+            return nil
+        }
+
+        guard let durationInMs = attachment.initialMetadata?.duration else {
```

Comment:
nitpick: In swift the standard naming would be `durationInMS`
```suggestion
        guard let durationInMS = attachment.initialMetadata?.duration else {
```

Reply:
super-duper nitpick: abbreviations are discouraged. suggested change: durationInMilliseconds